src/App.js

```diff
@@ -61,37 +64,27 @@ const Transition = styled(CSSTransition)`
   }
 
   &.page-fade-enter.page-fade-enter-active {
-      opacity: 1;
-      transition: opacity 300ms ease-in;
+    opacity: 1;
+    transition: opacity 300ms ease-in;
   }
 
   &.page-fade-exit {
-      display: none;
+    display: none;
   }
 `;
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
+const App = (props) => {
+  const [overlay, setOverlay] = useState(false);
 
-    this.loadUserInfo();
-
-    this.onMenuToggle = this.onMenuToggle.bind(this);
-
-    this.state = {
-      overlay: false,
-    };
-  }
-
-  loadUserInfo() {
+  const loadUserInfo = () => {
     // Get user details from localStorage and save to react store
     try {
-      var info = localStorage.getItem("userInfo");
+      let info = localStorage.getItem("userInfo");
       if (info) {
         let userInfo = JSON.parse(info);
         console.log("Loaded UserId from storage : " + userInfo.userId);
         console.log("Loaded token from storage : " + userInfo.token);
-        this.props.setCurrentUser({
+        props.setCurrentUser({
           userId: userInfo.userId,
           token: userInfo.token,
         });
```

Comment:
would be better if we just destructure the user info and use it to set the current user . use const instead of var or let

Reply:
`    try {
      const info = localStorage.getItem("userInfo");
      if (info) {
        const {userId , token) = JSON.parse(info);
        console.log("Loaded UserId from storage : " + userInfo.userId);
        console.log("Loaded token from storage : " + userInfo.token);
        this.props.setCurrentUser({
        props.setCurrentUser({
          userId,
          token
        });`